.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -12,3 +12,5 @@ contact_links:
   - name: Security Issues
     url: https://hackerone.com/bitwarden
     about: We use HackerOne to manage security disclosures.
+  - name: Report mobile autofill failure
+    url: https://docs.google.com/forms/d/e/1FAIpQLScMopHyN7KGJs8hW562VTzbIGL4KcFnx0wJcsW0GYE1BnPiGA/viewform
```

Comment:
🤔 Even though users can provide the information in `Any Additional Info?`, could we add an optional field in the form for the user to complete which field they were trying to autofill? or the operation they were trying to do? Like username, password, create passkey, autofill passkey, autofill TOTP, etc.

Reply:
@fedemkr definitely, we can ask our CS team!